screeps-game-api/src/game.rs

```diff
@@ -170,12 +172,185 @@ pub mod map {
 
     /// See [http://docs.screeps.com/api/#Game.map.isRoomAvailable]
     ///
-    /// [http://docs.screeps.com/api/#Game.map.isRoomAvailable]: http://docs.screeps.com/api/#Game.map.isRoomAvailable
+    /// [http://docs.screeps.com/api/#Game.map.isRoomAvaile]: http://docs.screeps.com/api/#Game.map.isRoomAvaile
     pub fn is_room_available(room_name: &str) -> bool {
         js_unwrap!(Game.map.isRoomAvailable(@{room_name}))
     }
+
+    /// Implements `Game.map.findExit`.
+    /// 
+    /// Does not yet support callbacks.
+    pub fn find_exit(from_room: Room, to_room: Room) -> Result<Exit, ReturnCode> {
+        let code: i32 = js_unwrap!{Game.map.findExit(@{from_room.name()}, @{to_room.name()})};
+        Exit::try_from(code).map_err(|v| v.try_into().expect("find_exit: Error code not recognized."))
+    }
+
+    #[allow(unused_variables)]
+    pub fn find_route(from_room: Room, to_room: Room, route_callback: Option<impl Fn(&str, &str) -> u32>) -> !{
+        unimplemented!()
```

Comment:
I'd rather leave these methods out for the sake of having compile-time errors rather than runtime ones when trying to use them.

We could leave it in here commented out for the sake of keeping the signature, though.

Thoughts?

Reply:
I like having them in the code because of the documentation generated (granted, this one doesn't have a doc).
I believed that using a function with the never return type wouldn't compile. After testing... it's annoying and confusing, so commenting it out seems like the best option.

Reply:
We could also have an `unimplemented-functions` feature which we enable when building docs but disable otherwise.

Or we could also change the function to accept a never/void input parameter '_unimplemented'! Never type in the return position can actually be called, it just guarantees the function will only ever panic or loop forever. If we include something like `enum UnimplementedParameterVoid {}` and then take `_unimplemented: UnimplementedParameterVoid` that would prevent anyone from calling it since no one would be able to construct that value.